tests/console/suseconnect_scc.pm

```diff
@@ -24,16 +24,18 @@ use base 'y2logsstep';
 
 use testapi;
 use utils 'zypper_call';
+use Utils::Backends 'use_ssh_serial_console';
```

Comment:
IMHO you're just adding but not using `use_ssh_serial_console()`, right? 
BTW I have in my TOOD list adding  `use_ssh_serial_console()` to `select_serial_terminal()` as often I see:
`check_var("BACKEND", "ipmi") ? use_ssh_serial_console : select_console 'root-console';`
But in case of these machines it looks it's not needed.

Reply:
Valid point. I'll remove this line